turtlebot2_cartographer/launch/turtlebot_carto_lidar_2d.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from ament_index_python.packages import get_package_share_directory
+from launch.exit_handler import restart_exit_handler
+from ros2run.api import get_executable_path
+
+
+def launch(launch_descriptor, argv):
+    ld = launch_descriptor
+    package = 'turtlebot2_drivers'
+    ld.add_process(
+        cmd=[get_executable_path(package_name=package, executable_name='kobuki_node')],
+        name='kobuki_node',
+        exit_handler=restart_exit_handler,
+    )
+    package = 'urg_node'
```

Comment:
There's no real reason to add the node to the .repos file -- we probably won't be adding anything to the .repos file.

But the repo doesn't need to be in the .repos file to be useful! People know about the port, and the info is available on discourse. What I'd recommend is either (a) posting the config file on discourse or (b) hosting a gist with the config file you can link to.